### ISSUE TITLE
🎉NEW anti-anti-adblock project - this one is inactive

### ISSUE THREAD

**Issue by ghost** [reek/anti-adblock-killer#4637]
Hello!

I've started a new project, it's called AdCrunch. 

For those interested in using AdBlock / Adblock Plus with a better user experience: https://raw.githubusercontent.com/AdCrunchSoftware/AdblockingFilters/main/antiantiadblock.txt

If you're using uBlock Origin: https://raw.githubusercontent.com/AdCrunchSoftware/AdblockingFilters/main/foruBO.txt

Please report anti-adblock issues here: https://github.com/AdCrunchSoftware/AdblockingFilters/issues

**Comment by garry-ut99** [comment]
> If you're using uBlock Origin

I don't see a point, because [uBlock Origin](https://github.com/gorhill/uBlock) and [AdGuard](https://adguard.com/en/adguard-browser-extension/overview.html)  ad-blockers already have their own anti-adblock built-in filter lists, where anti-adblock messages can be reported and will be blocked by filter list maintentainers:
- uBlock Origin filters issue tracker : https://github.com/uBlockOrigin/uAssets
- AdGuard filters issue tracker: https://github.com/AdguardTeam/AdguardFilters

Also there already exists an additional anti-adblock filter list: https://github.com/bogachenko/fuckfuckadblock
(though seems somewhat controversial: https://github.com/reek/anti-adblock-killer/wiki, and problematic in uBlock)

**Comment by ghost** [comment]
I know, I've been contributing to uBO and AdGuard extensively for a few years. But there still are different approaches that could be taken when dealing with a smaller number of users - megaup & uBlock Origin volunteers playing a cat-and-mouse anti-adblock game, whereas @@||megaup.net^$ehide would work fine for personal use

**Comment by garry-ut99** [comment]
Ok, but I wonder why to reinvent the wheel from scratch (means: why to create your list from scratch) ? Why not just to fork bogachenko's list and remove all controversial `@@` network exception rules or any unwanted rules, seems to be just easier and faster to adapt the existing list than to create a new one, unless you have a really different approach, or maybe you want to avoid issues of his list: https://github.com/bogachenko/fuckfuckadblock/wiki/Support-policy#known-issues

**Comment by ghost** [comment]
most of the entries are for AdBlock, and disregarded by uBO

**Comment by garry-ut99** [comment]
I'm speaking about version for uBlock https://raw.githubusercontent.com/AdCrunchSoftware/AdblockingFilters/main/foruBO.txt

**Comment by ghost** [comment]
uBlock Origin already does an excellent job at blocking ads and addressing anti-adblock, so there isn't much to add

**Comment by garry-ut99** [comment]
So why bogachenko's list has over 22000 filters, do you mean his list is obsolete now in uBlock?

**Comment by ghost** [comment]
Yes, mostly. It's banned by uBO anyway. 

Edit: I guess a better description of the opinion would be "mostly obsolete"

**Comment by bogachenkove** [comment]
> Yes, mostly. It's banned by uBO anyway.

@SeamlessRelaxation Yep, the Frenchman **loves** monopoly. Many of the rules that worked wrong were found by the uBlock Origin team, but were not reported to me. It got to the point that any mention on Reddit by a user about my list was punished, in the best case, by deleting the comment, in the worst case, by a ban. While I was trying to eliminate all the false positives found in the uBlock Origin sheet and reported it to the Japanese-guy - `Yuki2718` using ping, mail and PM on Reddit.
You'll find out soon enough (if your list gets popular enough) something that is not enabled by default in the uBlock Origin extension settings will be perceived as hostile by the uBlock team.

>So why bogachenko's list has over 22000 rules, do you mean his list is obsolete now in uBlock?

@garry-ut99 At the moment I am busy in real life, but I still try to maintain my project, 80 percent of the rules still work correctly, by the way, I recently created [tools](https://github.com/bogachenko/fuckfuckadblock/tree/master/.github/TOOLS) to maintain this list, I will improve them as much as possible.

***
Have a nice day.

**Comment by ghost** [comment]
@bogachenko I suppose an easy "fix" could be to disable the filters that generically allow google stuff for uBO users - they don't help uBO users that much either way and for ima3.js there's the `google-ima.js` scriptlet

```adblock
!#if !ext_ublock
!#endif
```

Have a nice day as well!

**Comment by garry-ut99** [comment]
> Seamless : It's banned by uBO anyway.

Unrelated to what I am speaking about: https://github.com/reek/anti-adblock-killer/issues/4637#issuecomment-2209490533,  you've again misunderstood the context of my question, I proposed you to fork his list and remove or disable filters which were a reason of the ban.

**Comment by PheKe** [comment]
bro can you give me the "foruBO.txt" file
i find it pretty good for bypassing aternos's adblock detect

**Comment by ajaxnotmeta** [comment]
> Hello!
> 
> I've started a new project, it's called AdCrunch.
> 
> For those interested in using AdBlock / Adblock Plus with a better user experience: https://raw.githubusercontent.com/AdCrunchSoftware/AdblockingFilters/main/antiantiadblock.txt
> 
> If you're using uBlock Origin: https://raw.githubusercontent.com/AdCrunchSoftware/AdblockingFilters/main/foruBO.txt
> 
> Please report anti-adblock issues here: https://github.com/AdCrunchSoftware/AdblockingFilters/issues

i tried to wonder your own issues repo, but it was 404 instead

**Comment by deltaxsingh** [comment]
Same as the user above, can't find the repo.

I switched from the adguard android app to the DNS because the app was so performance heavy. Having an anti-adblock userscript would be immensely helpful

**Comment by swathigrate** [comment]
> Same as the user above, can't find the repo.
> 
> I switched from the adguard android app to the DNS because the app was so performance heavy. Having an anti-adblock userscript would be immensely helpful

Yes.... Not getting that repo

**Comment by Draguljche** [comment]
[bogachenko/fuckfuckadblock](https://github.com/bogachenko/fuckfuckadblock) is dead to, repo is archived so if there is alternative or fork let me know.

**Comment by TomerGamerTV** [comment]
same
